<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>401</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>622</width>
    <height>401</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>622</width>
    <height>401</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift Light</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>PTZ Control</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>48</width>
    <height>48</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>PT Movements</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>251</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QPushButton" name="noneButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string>This button is used to stop movement (It will send the PELCO-D Stop message)</string>
        </property>
        <property name="whatsThis">
         <string>This button is used to stop movement (It will send the PELCO-D Stop message)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="media-record"/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="rightButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::GoNext"/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="downButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::GoDown"/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="upButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::GoUp"/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="leftButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::GoPrevious"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="keyboardCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="statusTip">
      <string>This option enables the Keyboard control to control PT via keyboard instead of buttons</string>
     </property>
     <property name="text">
      <string>&amp;Keyboard Control</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>241</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Created Pelco Command</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="statusTip">
      <string>The Logs will be displayed here when the connection has been established.</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Connection Status</string>
    </property>
    <widget class="QLabel" name="statusLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Not Connected!</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>231</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="PortComboBox">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&amp;Port:</string>
        </property>
        <property name="buddy">
         <cstring>PortComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&amp;Baudrate:</string>
        </property>
        <property name="buddy">
         <cstring>BaudrateComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="BaudrateComboBox">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>230400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>460800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>921600</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Set Limits</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Set Left Limit (Degree°)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Set Right Limit (Degree°)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leftLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="rightLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="presetEnableCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="statusTip">
      <string>This option enables the PRESET Functionality to set-reset limits.</string>
     </property>
     <property name="text">
      <string>P&amp;RESET functionality</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Focus/Zoom</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>12</y>
       <width>221</width>
       <height>83</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QPushButton" name="zoomInButton">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Zoom in</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="focusOutButton">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Focus far</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="zoomOutButton">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Zoom out</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="focusInButton">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Focus near</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="stopAllButton">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string>This button is used to stop movement (It will send the PELCO-D Stop message)</string>
        </property>
        <property name="whatsThis">
         <string>This button is used to stop movement (It will send the PELCO-D Stop message)</string>
        </property>
        <property name="text">
         <string>Stop Movement</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Please click if you want to connect the remote PT device with selected Port-Baudrate Configuration</string>
       </property>
       <property name="statusTip">
        <string>Please click if you want to connect the remote PT device with selected Port-Baudrate Configuration</string>
       </property>
       <property name="whatsThis">
        <string>Please click if you want to connect the remote PT device with selected Port-Baudrate Configuration</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>START</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::DocumentSend"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Please click if you want to disconnect the remote PT device.</string>
       </property>
       <property name="statusTip">
        <string>Please click if you want to disconnect the remote PT device.</string>
       </property>
       <property name="text">
        <string>STOP</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>622</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_Config_As"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionSerial_Port"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAdvanced_View"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionClear_cache"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionCheck_Updates"/>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save Config</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>A&amp;bout</string>
   </property>
  </action>
  <action name="actionCheck_Updates">
   <property name="text">
    <string>Check &amp;Updates</string>
   </property>
  </action>
  <action name="actionClear_cache">
   <property name="text">
    <string>&amp;Clear cache</string>
   </property>
  </action>
  <action name="actionAdvanced_View">
   <property name="text">
    <string>A&amp;dvanced View</string>
   </property>
  </action>
  <action name="actionSerial_Port">
   <property name="text">
    <string>Serial &amp;Port Advanced</string>
   </property>
  </action>
  <action name="actionSave_Config_As">
   <property name="text">
    <string>Save Config &amp;As..</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>PTZ Control &amp;Help</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open..</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>PortComboBox</tabstop>
  <tabstop>BaudrateComboBox</tabstop>
  <tabstop>upButton</tabstop>
  <tabstop>noneButton</tabstop>
  <tabstop>leftButton</tabstop>
  <tabstop>rightButton</tabstop>
  <tabstop>downButton</tabstop>
  <tabstop>keyboardCheckBox</tabstop>
  <tabstop>presetEnableCheckBox</tabstop>
  <tabstop>leftLineEdit</tabstop>
  <tabstop>rightLineEdit</tabstop>
  <tabstop>stopAllButton</tabstop>
  <tabstop>zoomInButton</tabstop>
  <tabstop>zoomOutButton</tabstop>
  <tabstop>focusInButton</tabstop>
  <tabstop>focusOutButton</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
